pkg/trace/stats/quantile/ddsketch_converter.go

```diff
@@ -0,0 +1,180 @@
+package quantile
+
+import (
+	"errors"
```

Comment:
Can you please sort the imports into groups like the rest of the source? We use 3 groups:

1. Stdlib
2. Local (datadog-agent) imports
3. External